.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+# Plugin Version : X.X.X (next version number)
+
+**Version changelog**
+
+A list of changes regarding the next version release:
+
+1. Short description of the new feature/fix - #Issue/PR-Number
+2. Short description of the new feature/fix - #Issue/PR-Number
+
+**Release checklist**
+
+- [ ] Added `release` tag to this pull request
+- [ ] Updated README.md
+- [ ] Updated CHANGELOG.md
+- [ ] Updated composer.json
+- [ ] Updated plugin version number
+- [ ] Updated screenshots in assets folder
+- [ ] Updated translations
+- [ ] Updated userguide.md
+- [ ] Tested module with [validation tool](https://github.com/sendsmaily/smaily-magento-extension/blob/master/CONTRIBUTING.md#magento-2-module-package-testing)
+- [ ] Tested [compiling in production mode](https://github.com/sendsmaily/smaily-magento-extension/blob/master/CONTRIBUTING.md#compilation-and-production-mode-testing)
+
+**After successful review**
+
+- [ ] [Packaged module](https://github.com/sendsmaily/smaily-magento-extension/blob/master/CONTRIBUTING.md#checklist-for-magento-marketplace) and submitted to Magento marketplace
+
```

Comment:
This packaging should be still done manually.

We could use the same workflow as in other plugins to generate a zip file after release, but that happens after acceptance in the Magento marketplace.